.vscode/settings.json

```diff
@@ -16,5 +18,23 @@
     ],
     "python.analysis.extraPaths": [
         "/vendor"
-    ]
-}
+    ],
+    "files.associations": {
+        "*.html": "jinja-html"
+    },
+    "emmet.includeLanguages": {
+        "jinja-html": "html"
+    },
+    "prettier.disableLanguages": [
+        "jinja-html"
+    ],
+    "beautify.language": {
+        "html": [
+            "jinja-html"
+        ]
+    },
+    "[jinja-html]": {
+        "editor.formatOnSave": true,
+        "editor.defaultFormatter": "HookyQR.beautify"
+    }
+}
```

Comment:
Is there a vscode setting we can add to get it to add newlines to the ends of files (like this one).

Reply:
Added it in the settings